packages/core/src/tools.ts

```diff
@@ -150,16 +150,16 @@ export namespace Tools {
 	}
 
 	/**
-	 * Get the percentage of a datapoint compared to the entire data-set.
-	 * Returns 1 significant digit if percentage is less than 1%.
+	 * Get the percentage of a datapoint compared to the entire dataset.
+	 * Returns 1 significant digit.
 	 * @export
 	 * @param {any} item
 	 * @param {any} fullData
 	 * @returns The percentage in the form of a number
 	 */
 	export function convertValueToPercentage(item, fullData) {
 		const percentage = item / fullData.reduce((accum, val) => accum + val.value, 0) * 100;
-		return percentage < 1 ? percentage.toPrecision(1) : Math.floor(percentage);
+		return percentage.toFixed(1);
```

Comment:
```suggestion
		// if the value has any significant figures, keep 1
		return percentage % 1 !== 0 ? parseFloat(percentage.toFixed(1)) : percentage;
```

Reply:
good call, a .0 wouldnt be more informative